orbotservice/src/main/java/org/torproject/android/service/TorServiceConstants.java

```diff
@@ -9,14 +9,6 @@
 
     String DIRECTORY_TOR_DATA = "tordata";
 
-    String TOR_CONTROL_PORT_FILE = "control.txt";
-    String TOR_PID_FILE = "torpid";
-
-    //torrc (tor config file)
-    String TORRC_ASSET_KEY = "torrc";
-
-    String TOR_CONTROL_COOKIE = "control_auth_cookie";
-
     //geoip data file asset key
     String GEOIP_ASSET_KEY = "geoip";
```

Comment:
are these still used?

Reply:
Looks like one of them is a duplicate

Reply:
Both of those constants were there already.  I updated this pull request to remove one